gems/sorbet-runtime/test/types/method_validation.rb

```diff
@@ -373,14 +373,14 @@ def @mod.foo(req_str, opt_hash={}, kwopt_bool: false)
             if signature.on_failure
               T::Configuration.soft_assert_handler(
                 "TypeError: #{opts[:pretty_message]}",
-                {notify: signature.on_failure[0][:notify]}
+                {notify: signature.on_failure[1][:notify]}
```

Comment:
huh?

Reply:
it's because i changed the `on_failure` arguments in our test case below:

https://github.com/sorbet/sorbet/pull/1176/files#diff-c68234159e068097095fb22edbed4977R383

I asked about whether people wanted `.on_failure(notify: ...)` or `.on_failure(:soft, notify: ...)` in #ruby and they said they wanted this, because it made it more explicit that it was turning sig failure into a soft assertion.